STL_Extension/test/STL_Extension/test_Object.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <CGAL/Object.h>
 #include <CGAL/assertions.h>
+#include <CGAL/use.h>
```

Comment:
> How did that get included before??? 

It is included by `<CGAL/Algebraic_structure_traits.h>`, that is one of the dependencies of `<CGAL/basic.h>`.